dashboard.csss-ci.com/server/api/controllers/rest/v1/Quotations.php

```diff
@@ -6,12 +6,23 @@
 
 require APPPATH . '/libraries/REST_Controller.php';
 
+/**
+ * Class for Quotations API
+ */
 class Quotations extends REST_Controller {
 
+    /**
+     * Invokes the parent class's constructor 
+     */
     public function __construct() {
         parent::__construct();
     }
 
+    /**
+     * Gets the index of the proposal
```

Comment:
Needs more context, more detail. This does not make any sense, what do you mean by gets the index of the proposal?